### PR TITLE
Tickets/sitcom 1444

### PR DESCRIPTION
**Pull request by Kelks19**
I have migrated the procedure to Obs-Ops. I have also corrected a bit of English and the flow while keeping teh content exactly same. 
There is several important things  I identified as missing from the content --- 
1. Please note that there are two figures missing-- I think we need to create a follow-up ticket to add them once we get access to OSS activity from the TMA EUI 
2. We need to highlight again how to access to TMA EUI  and include link to the relevant page 
3. The confluence page says that *Safety Team* should be included as authors. Can you specifiy which ones ? 

I was not involved in the content creation and the review of the original Confluence page.

### CODE REVIEW

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 1]
```diff
@@ -70,20 +75,98 @@ Procedure Steps
 .. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
 .. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
 
+.. note::
```
Comment (isotuela) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:78]: Maybe change this note to the first step of the procedure. 1.  Before/right after applying LOTO on the OSS, announce "LOTO on the OSS" on  *#summit-announce* and/or *#summit-simonyi*.

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 2]
```diff
@@ -70,20 +75,98 @@ Procedure Steps
...
 
+    Before/right after applying LOTO on the OSS, announce "LOTO on the OSS" on  *#summit-announce* and/or *#summit-simonyi*
```
Comment (isotuela) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:80]: Needs a . to finish sentence.

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 3]
```diff
@@ -70,20 +75,98 @@ Procedure Steps
...
 
-.. _Simonyi-LOTO-procedure-Troubleshooting:
 
-Troubleshooting
-===============
+Applying LOTO on OSS
+--------------------
+
+#.  Turn :guilabel:`OFF` the OSS in the EUI panel (indicated by the :gold:`circle` in the following Figure)
```
Comment (isotuela) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:86]: Figure doesn´t need to be capitalized.

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 4]
```diff
@@ -70,20 +75,98 @@ Procedure Steps
...
+
+    .. figure:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-1.png
+     :name: tma-loto-1
```
Comment (isotuela) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:89]: Please add caption to figures.

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst (hunk 3 above)
Comment (isotuela) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:86]: Bold a summary of each step like step 2.

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 5]
```diff
@@ -70,20 +75,98 @@ Procedure Steps
...
+
+#.  **Wait until the OSS is off**:  the :guilabel:`status` turns `IDLE` and :guilabel:`OSS` flag goes from green to gray. 
+    It takes around 5-10 minutes to complete turning off the OSS. 
+
+    .. figure:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-2.png
+     :name: tma-loto-2
```
Comment (isotuela) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:95]: Caption

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 6]
```diff
@@ -70,20 +75,98 @@ Procedure Steps
...
+
+#.  Go to the OSS room from level 1 (first floor). Apply LOTOs on the OSS pump switches (highlighted by the :red:`box` in
+    the following Figure). Close valves (Position "—-") and put locks on the holes to apply three LOTOs on the safety 
+    switches of the OSS pumps.
+
+    .. figure:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-3.png
+     :name: tma-loto-3
```
Comment (isotuela) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:102]: Add caption to figure.

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 7]
```diff
@@ -70,20 +75,98 @@ Procedure Steps
...
+     :width: 30%
+    .. image:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-6.png
+     :width: 30%
+
+.. note::
+
+    **LOTO box**
+    
+    Keys and locks are inside the LOTO Box. You can find LOTO box (with a tag "Electronic team" on the box) on 
+    Level 1 in the OSS room. If you cannot find or open that LOTO box, you can take a LOTO box in the meeting room
```
Comment (isotuela) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:116]: you can take a LOTO box FROM the meeting room

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 8]
```diff
@@ -70,20 +75,98 @@ Procedure Steps
...
+    .. figure:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-1.png
+     :name: tma-loto-1
+
+#.  **Wait until the OSS is off**:  the :guilabel:`status` turns `IDLE` and :guilabel:`OSS` flag goes from green to gray. 
+    It takes around 5-10 minutes to complete turning off the OSS. 
+
+    .. figure:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-2.png
+     :name: tma-loto-2
+
+#.  Go to the OSS room from level 1 (first floor). Apply LOTOs on the OSS pump switches (highlighted by the :red:`box` in
```
Comment (isotuela) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:97]: Bold in summary of step *Apply LOTOs on the OSS pump switches*

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 9]
```diff
@@ -70,20 +75,98 @@ Procedure Steps
...
+    -   OSS turned on after removing LOTO  (:red:`Need Figure`)
 
-No troubleshooting information is applicable to this procedure.
-In the IMPOSSIBLE event that the telescope moves after turning off all three breakers, stop work immediately and notify the day crew, including Jacques Sebag, Mario Rivera, and the electronics team. 
+.. _Simonyi-LOTO-procedure-Contingency:
 
+Contingency
+===========
 
+If the procedure was not successful, report the issue in *#summit-simonyi* and/or *#summit-announce* and/or activate
```
Comment (isotuela) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:169]: Not sure how this procedure can be unsuccesful. Perhaps reword it?

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 10]
```diff
@@ -70,20 +75,98 @@ Procedure Steps
...
+    .. image:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-4.png
+     :width: 30%
+    .. image:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-5.png
+     :width: 30%
+    .. image:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-6.png
+     :width: 30%
+
+.. note::
+
+    **LOTO box**
```
Comment (isotuela) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:113]: Consider moving this LOTO box note before figures to line 100 or before first step (line 77) so it applies to Lock and Unlock procedures.

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 11]
```diff
@@ -70,20 +75,98 @@ Procedure Steps
...
-===============
+Applying LOTO on OSS
+--------------------
+
+#.  Turn :guilabel:`OFF` the OSS in the EUI panel (indicated by the :gold:`circle` in the following Figure)
+
+    .. figure:: /Simonyi/Non-Standard-Operations/MTCS/TMA/_static/tma-loto-1.png
+     :name: tma-loto-1
+
+#.  **Wait until the OSS is off**:  the :guilabel:`status` turns `IDLE` and :guilabel:`OSS` flag goes from green to gray.
```
Comment (isotuela) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:91]: Idle not in caps, just how it shows in the EUI.

**Review by isotuela — CHANGES_REQUESTED** [review]
Thank you! It looks good. I have added some suggestions, mostly regarding the caption in figures and the position of the LOTO box note. I think if each step's key words is in bold, it would improve reading, just like in step 2 of Applying LOTO on OSS. 
Also, please squash commits into a single one. Let me know if you need any help! 
Thank you!

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst (hunk 1 above)
Comment (Kelks19) [reply Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:78]: done

**Review by isotuela — APPROVED** [review]

**Review by Slimleashma — COMMENTED** [review]
Thank you for the new photos and explicit instructions about the LOTO boxes. One note to make (and maybe there is hesitance to make this official) is that during the night, only ONE pump needs to be locked to consider the TMA fully under LOTO. This is helpful when one cannot find the LOTO boxes or there aren't multiple people around to move this process faster. It's something to note that LOTO at night with multiple locks and keys raises a risk of leaving lights on longer and losing these multiples. Can we consider this additional caveat for the document?

**Comment by Kelks19** [comment]
@Slimleashma I can add a note box with  'LOTO at night ' to highlight these caveats -- What do you think ? 
This note can be convereted to sub-section after we confirm the official word on it.

**Comment by Slimleashma** [comment]
That could be good. Multiple locks is overkill when we are talking about a
two-person operation at night.

Alysha

File: Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst [hunk 12]
```diff
@@ -47,19 +61,10 @@ Precondition
...
-
-.. This section should provide a simple overview of conditions or results after executing the procedure; for example, state of equipment or resulting data products.
-.. It is preferred to include them as a bulleted or enumerated list.
-.. Please provide screenshots of the software status or relevant display windows to confirm.
-.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
-
-The drives are powered off and the AuxTel Control cabinet is locked from the outside. 
-The Simonyi Telescope is under LOTO and is not able to move by any controls.
+    To perform this procedure formal LOTO training must have been received; refer also to 
+    the `TMA LOTO Procedure video <https://drive.google.com/file/d/1rOvu1ITDCm0HNepfvoBWMzhGZkMPkKnH/view>`_
```
Comment (isotuela) [inline Simonyi/Non-Standard-Operations/MTCS/TMA/Simonyi-LOTO-procedure.rst:67]: . at the end of the sentence.

**Review by isotuela — CHANGES_REQUESTED** [review]
Only one minor grammar comment. Could you please squash your commits into one? Let me know if you encounter any issue. 

Thanks! 
Ioana

**Review by Slimleashma — APPROVED** [review]
Looks great! Thanks for the nighttime caveat.

**Review by isotuela — APPROVED** [review]
All good and ready to merge.
Thanks!

**Pull request merged**